src/vs/platform/quickinput/browser/quickInput.ts

```diff
@@ -1749,6 +1751,7 @@ export class QuickInputController extends Disposable {
 		ui.description1.style.display = visibilities.description && (visibilities.inputBox || visibilities.checkAll) ? '' : 'none';
 		ui.description2.style.display = visibilities.description && !(visibilities.inputBox || visibilities.checkAll) ? '' : 'none';
 		ui.checkAll.style.display = visibilities.checkAll ? '' : 'none';
+		ui.inputContainer.style.display = visibilities.inputBox ? '' : 'none';
```

Comment:
cc @TylerLeonhardt

Reply:
What does this fix?

Reply:
<img width="656" alt="image" src="https://github.com/microsoft/vscode/assets/10746682/778e4b84-675e-4a25-bb1d-558484c2d302">